src/views/Document.vue

```diff
@@ -41,13 +38,19 @@
 import { mapGetters } from 'vuex';
 import ResultHeader from '../components/Result/ResultHeader';
 import Toggle from '@/components/Global/Toggle/Toggle';
+import { prevRoute } from '../router/index.js';
 
 export default {
 	components: {
 		ResultHeader,
 		Toggle
 	},
 	computed: {
+		backToPrev: function () {
```

Comment:
NIT: `getGoBackURL` or sth like that? Cuz `backToPrev` sounds like a procedure that does something to me, but in reality, it just returns route object.